src/lib/modules/blockchain_process/httpProxyOverride.js

```diff
@@ -0,0 +1,123 @@
+/* global require */
+
+const http = require('http');
+const https = require('https');
+const httpProxyWsIncoming = require('http-proxy/lib/http-proxy/passes/ws-incoming');
+const common = require('http-proxy/lib/http-proxy/common');
+
+const CRLF = '\r\n';
+
+httpProxyWsIncoming.stream = (req, socket, options, head, server, cb) => {
+  const createHttpHeader = function(line, headers) {
```

Comment:
I believe this function can be defined outside of `stream` no?

Also, if we go with function initialization instead of declaration, can we use arrow functions instead?

Reply:
`httpProxyOverride.js` was copied from an unmerged PR for the `http-proxy` package:

https://github.com/nodejitsu/node-http-proxy/pull/1301

We're using it to patch `http-proxy` with support for transform streams.

Some light cleanup was done to the formatting of the copied code, but it wasn't otherwise modified. We certainly can introduce additional changes, but more time will need to be spent studying the source to make sure we don't break it.